routes/dao_coin_exchange.go

```diff
@@ -18,6 +19,13 @@ import (
 type GetDAOCoinLimitOrdersRequest struct {
 	DAOCoin1CreatorPublicKeyBase58Check string `safeForLogging:"true"`
 	DAOCoin2CreatorPublicKeyBase58Check string `safeForLogging:"true"`
+
+	// A list of hex OrderIds that we will fetch
+	OrderIds []string `safeForLogging:"true"`
```

Comment:
+1 to Nina's comment. Please create a new endpoint that takes in `OrderIds` and returns the exact orders. Overloading the existing endpoint with more orthogonal fields will make its usage more error prone.